### DISCUSSION TITLE
Has VpnHood been audited?

### DISCUSSION THREAD

**Discussion by mrphs** [vpnhood/VpnHood#222] [Q&A] [answered]
I was wondering if VpnHood has gone through any independent security audit and if the report has been published anywhere public?

**Comment by bobvhood** [comment]
No, Do you know any company that can do that for an open-source project without cost?

**Reply by mrphs** [reply]
Actually, I do! Check out https://www.opentech.fund/labs/red-team-lab/

**Reply by trudyhood** [reply, marked as answer]
We got this: They reported 3 minor optional considerations required for supporting the older version of Android, and two of them have already been resolved.

https://www.opentech.fund/security-safety-audits/vpnhood-security-audit-results/

Please let us know if you have any more ideas for helping the VpnHood community.
